diff --git a/vllm/config/profiler.py b/vllm/config/profiler.py
index b3b232490f..e40baa9ac5 100644
--- a/vllm/config/profiler.py
+++ b/vllm/config/profiler.py
@@ -97,6 +97,10 @@ class ProfilerConfig:
     """If `True`, enables memory profiling in the torch profiler.
     Disabled by default."""
 
+    torch_profiler_summary_only: bool = False
+    """If `True`, writes aggregated operator tables instead of a trace.
+    Disabled by default."""
+
     capture_torch_profiler: bool = False
     """If `True`, enables a torch profiler during CUDA graph capture on rank 0.
     Traces are saved to a `capture_traces` subdirectory under `torch_profiler_dir`.
